### DISCUSSION TITLE
Request: Add ability to donate to the project

### DISCUSSION THREAD

**Discussion by richid** [arabcoders/watchstate#346] [General] [answered]
👋 I've looked around (didn't check Discoard, ftr) and I haven't found a way to donate a few bucks to you. This is a great little project that has served me very well in migrating from Plex/Emby over to Jellyfin and I would love to make a small donation in appreciation and support of the project.

Maybe adding one of those [Buy me a coffee buttons](https://www.buymeacoffee.com/) would be a start?

Thank you!

**Comment by arabcoders** [comment, marked as answer]
Hi,

I am glad the tool is useful for you. I do not accept donation as i am developing this tool for myself first. If you feel like donating please donate to any charity that takes care of kids everywhere. 🌹
